components/app/account/AccountHeader.vue

```diff
@@ -29,7 +29,23 @@
 
     <div>
       <div class="shadow rounded">
-        <h6 class="text-primary pt-4 pl-4 pb-2">{{ $t('manage.admins') }}</h6>
+        <div class="flex justify-between items-center px-4">
+          <label class="label">
+            <span class="label-text text-primary">{{
```

Comment:
suggestion(blocking): i think the classes of the span can also be moved into the label. We could omit the span this way.